sdk/tests/wallet/error.rs

```diff
@@ -6,12 +6,6 @@ use pretty_assertions::assert_eq;
 
 #[test]
 fn stringified_error() {
-    let error = Error::AccountNotFound("0".into());
-    assert_eq!(
-        &serde_json::to_string(&error).unwrap(),
-        "{\"type\":\"accountNotFound\",\"error\":\"account 0 not found\"}"
-    );
-
```

Comment:
Don't simply remove them, these were testing specific serialisations. Please replace by other errors with similar patterns. One tuple error, one struct error and one unit error.

Reply:
not sure I follow tbh ... it's not possible anymore that an account (or a wallet) cannot be found (which is why I removed the error variant).  So what should I replace it with?

Reply:
This has nothing to do with the actual variant, we're just testing error serialization here. Just pick a random other variant that has the same structure.

Reply:
We just need to test one tuple, one struct and one unit

Reply:
lol, I see

Reply:
Added a test for each kind.

Reply:
I don't see them

Reply:
oh, I changed the test in the bindings, not this one, should be fixed now